src/repository.cpp

```diff
@@ -89,24 +89,27 @@ repository_t::~repository_t() {
 }
 
 void
-repository_t::load(const std::string& path) {
-    const auto status = fs::status(path);
+repository_t::load(const std::vector<std::string>& plugin_dirs) {
+    COCAINE_LOG_INFO(m_log, "loading plugins");
+    std::vector<std::string> paths;
+    for (const auto& dir : plugin_dirs) {
+        const auto status = fs::status(dir);
 
-    if(!fs::exists(status) || !fs::is_directory(status)) {
-        COCAINE_LOG_ERROR(m_log, "unable to load plugins: path '{}' is not valid", path);
-        return;
-    }
+        if(!fs::exists(status) || !fs::is_directory(status)) {
+            COCAINE_LOG_WARNING(m_log, "loading plugins: path '{}' is not valid", dir);
```

Comment:
One can specify reasonable default folders and even one is missing at all it can be ok. Besides this is only a log line. If it is an unrecoverable error - we should throw an exception.